installconfig.adoc

```diff
@@ -79,6 +79,17 @@ endif::networkeditiondoc[]
 | Select the sub-menu for
   <<imapd_config,IMAPD configuration>>
 
+| zimbra-chat
+| Installing the Zimbra-Chat package is optional.
+  If you choose to install Zimbra-Chat for point-to-point text chat, it should
+  be installed on every Zimbra Store Server that is part of the Zimbra configuration.
+
+| zimbra-drive
+| Installing the Zimbra-Chat package is optional.
+  If you choose to install Zimbra-Chat for point-to-point text chat, it should
```

Comment:
Is this true for Zimbra-Drive true?  If so, update accordingly or replace with appropriate text.

Reply:
done